CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to TEAMMATES
+If you are just starting out, do refer to the [Contributor's Orientation Guide][COG].  
```

Comment:
I think this fits to be put right at the start (after organizing to two sections).

# Contributing to TEAMMATES

**Are you a new contributor?** Check out the Contributor's Orientation Guide.

### Submitting a bug report
...